### ISSUE TITLE
Improve logging - Error on first import 

### ISSUE THREAD

**Issue by sarmad-abualkaz** [yarmand/ragyva#12]
I ran the command 
`find ./docs -name '*.md' | ./import.py`

The output was:
```
Import filenames from stdin stdin
Error occurred: Table doc_model does not exist.Please first call db.create_table(doc_model, data). Creating table doc_model.
with 1 chunks 
.
```

I feel like this can be probably logged as a Warning instead - as this gives the impression the import had failed when in fact it created the table later. 

Let me know what you thinlk.

**Comment by yarmand** [comment]
yes, this exception handling [here](https://github.com/yarmand/ragyva/blob/1d61c63b93aaa457be39b0b3c46dd9b0ceefb26f/general/db.py#L20) is pretty poor.
I didn't get a chance to look into the type of exception lanceDB returns in the case of non existing table.
Proper way to do it would probably be to catch the exception when the table does not exist and log a warning.
And log an error for all other exceptions.

Also my logs are REALLY poor. I'm still learning python and didn't spent the time yet to select a proper loging library.

I'd love if you could take care of this 2 things  in 2 PRs ;)
